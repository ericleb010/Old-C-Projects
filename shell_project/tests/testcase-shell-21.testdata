sleep 10 &
sleep 15 &
