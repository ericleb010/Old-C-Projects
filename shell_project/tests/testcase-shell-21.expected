
------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 
